torch/_torch_docs.py

```diff
@@ -3099,7 +3099,7 @@ def merge_dicts(*dicts):
            r"""
 .. function:: max(input) -> Tensor
 
-Returns the maximum value of all elements in the :attr:`input` tensor.
+Returns the maximum value of all elements in the ``input`` tensor.
```

Comment:
Historically we've used :attr:`input` instead of ``input``. This is true for our other functions as well: https://github.com/pytorch/pytorch/blob/139aa519628b53f93af5538109f114d34823bf7f/torch/_torch_docs.py#L175-L179. I don't know what the difference is, but it would be good to be consistent with the rest of our docs.

Reply:
See https://github.com/pytorch/pytorch/pull/25885#pullrequestreview-285839021 . In this context `:attr:` should have the same effect as ` ``..`` `, because an attribute named `input` is missing in this context. If there is an `input` attribute or should there be one in the future, an issue similar to #25885 will surface.

The point is that we can always correct these in passing unless it starts to cause issues

Reply:
Got it. So it sounds like our other usages of `attr:` are incorrect as well, but we can fix those as we go along.